packages/inspector/src/components/Renderer/AxisHelper/AxisHelper.tsx

```diff
@@ -0,0 +1,223 @@
+import React, { useEffect, useRef } from 'react';
+import * as BABYLON from '@babylonjs/core';
+import { useSdk } from '../../../hooks/sdk/useSdk';
+import './AxisHelper.css';
+
+const AxisHelper: React.FC = () => {
```

Comment:
could you use the [withSdk HoC](https://github.com/decentraland/creator-hub/blob/main/packages/inspector/src/hoc/withSdk.tsx) for this component?